project/src/demo/puzzle/legacy-rank-result.gd

```diff
@@ -0,0 +1,127 @@
+class_name LegacyRankResult
+## Contains rank information for a playthrough. This includes raw statistics such as how many lines-per-minute the
+## player cleared, as well as derived statistics such as the computed lines-per-minute rank.
+##
+## Note: The math-based rank algorithms have been replaced with simpler percent-based algorithms, where there is a
+## perfect score and then other goals are calculated based on a percentage of the perfect score. The more granular
+## fields such as "box_score_per_line_rank" are no longer needed by these newer algorithms.
+
+const BEST_RANK := Ranks.BEST_RANK
+const WORST_RANK := Ranks.WORST_RANK
```

Comment:
These fields have been moved to RankResult elsewhere. We shouldn't need local copies.

Reply:
Removed BEST_RANK, WORST_RANK constants.